### PR TITLE
feat: Typeorm modification (MAPCO-4071 , MAPCO-4089 , MAPCO-4090)

### PR DESCRIPTION
**Pull request by RonitKissis**
| Question                | Answer                                                                          |
| ---------------- | -------------------------------------------------------------------------- |
| Bug fix         | ✖                                                                        |
| New feature     | ✔                                                                        |
| Breaking change | ✖                                                                        |
| Deprecations    | ✔                                                                       |
| Documentation   | ✖                                                                        |
| Tests added     | ✔                                                                      |
| Chore            | ✖                                                                       |

Further  information:

### CODE REVIEW

File: src/index.ts
Comment (netanelC) [inline src/index.ts:18]: Unnecessary

File: src/index.ts
Comment (netanelC) [inline src/index.ts:1]: Why did you add this?

File: tests/integration/heartbeat/heartbeat.spec.ts
Comment (netanelC) [inline tests/integration/heartbeat/heartbeat.spec.ts:76]: Remove this comment

File: tests/integration/heartbeat/heartbeat.spec.ts
Comment (netanelC) [inline tests/integration/heartbeat/heartbeat.spec.ts:51]: Remove this comment

File: tests/configurations/integration/jest.globalSetup.ts
Comment (netanelC) [inline tests/configurations/integration/jest.globalSetup.ts:12]: The schema will never be undefined

File: tests/configurations/integration/jest.globalTeardown.ts
Comment (netanelC) [inline tests/configurations/integration/jest.globalTeardown.ts:8]: The schema will never be undefined

File: README.md
Comment (netanelC) [inline README.md:90]: Unnecessary empty line

File: README.md
Comment (netanelC) [inline README.md:91]: Unnecessary empty line

File: dataSource.migrations.js
Comment (netanelC) [inline dataSource.migrations.js:10]: Is this necessary?

File: dataSource.migrations.js
Comment (netanelC) [inline dataSource.migrations.js:12]: Is this necessary?

**Review by netanelC — APPROVED** [review]
An impressive PR, keep up the hard work! most of the comments are lint issues so I won't block the PR but handle them before you're merging to the master branch

File: tests/configurations/integration/jest.globalSetup.ts
Comment (netanelC) [reply tests/configurations/integration/jest.globalSetup.ts:12]:
```suggestion
  const schema = dataSourceOptions.schema !== undefined ? `"${dataSourceOptions.schema}"` : 'public';
```

File: README.md
Comment (RonitKissis) [reply README.md:91]: thank you mister linter

File: src/DAL/utils/createConnection.ts
Comment (asafMasa) [inline src/DAL/utils/createConnection.ts:26]: is it possible to add log here?

File: src/index.ts
Comment (asafMasa) [inline src/index.ts:14]: is it possible to add log here of - "starting app"

File: tests/integration/heartbeat/heartbeat.spec.ts
Comment (asafMasa) [reply tests/integration/heartbeat/heartbeat.spec.ts:51]: revert remark keep code

File: tests/integration/heartbeat/heartbeat.spec.ts
Comment (asafMasa) [inline tests/integration/heartbeat/heartbeat.spec.ts:69]: add expect(response).toSatisfyApiSpec();

File: tests/integration/heartbeat/heartbeat.spec.ts
Comment (asafMasa) [reply tests/integration/heartbeat/heartbeat.spec.ts:76]: revert remark keep code

**Review by asafMasa — CHANGES_REQUESTED** [review]

File: src/index.ts
Comment (RonitKissis) [reply src/index.ts:18]: wrong, eslint throws error on /livenss

File: src/index.ts
Comment (RonitKissis) [reply src/index.ts:14]: what about line 20?

File: dataSource.migrations.js
Comment (RonitKissis) [reply dataSource.migrations.js:12]: removed

File: src/DAL/repositories/heartbeatRepository.ts
Comment (RonitKissis) [reply src/DAL/repositories/heartbeatRepository.ts:11]: cant log here but this class is called by the manager and there are logs there

File: src/DAL/repositories/heartbeatRepository.ts
Comment (RonitKissis) [reply src/DAL/repositories/heartbeatRepository.ts:26]: cant log here but this class is called by the manager and there are logs there

File: src/DAL/repositories/heartbeatRepository.ts
Comment (RonitKissis) [reply src/DAL/repositories/heartbeatRepository.ts:11]: log exists in heartbeatManager

File: src/DAL/repositories/heartbeatRepository.ts
Comment (RonitKissis) [reply src/DAL/repositories/heartbeatRepository.ts:26]: log exists in heartbeatManager

File: src/DAL/utils/createConnection.ts
Comment (RonitKissis) [reply src/DAL/utils/createConnection.ts:30]: added

File: tests/integration/heartbeat/heartbeat.spec.ts
Comment (RonitKissis) [reply tests/integration/heartbeat/heartbeat.spec.ts:51]: added a to do due to the bug we discussed

File: tests/integration/heartbeat/heartbeat.spec.ts
Comment (RonitKissis) [reply tests/integration/heartbeat/heartbeat.spec.ts:69]: added a to do due to the bug we discussed

File: tests/integration/heartbeat/heartbeat.spec.ts
Comment (RonitKissis) [reply tests/integration/heartbeat/heartbeat.spec.ts:77]: added a to do due to the bug we discussed

File: src/common/utils.ts
Comment (RonitKissis) [reply src/common/utils.ts:1]: can you explain the purpose of that? Heartbeat manager doesnt use mc-utils

**Review by asafMasa — CHANGES_REQUESTED** [review]
Good Job!

**Review by asafMasa — APPROVED** [review]

**Pull request merged**
